packages/exec/__tests__/exec.test.ts

```diff
@@ -286,6 +286,27 @@ describe('@actions/exec', () => {
     expect(stderrCalled).toBeTruthy()
   })
 
+  it('Handles stdin', async () => {
+    const waitForInput: string = path.join(
+      __dirname,
+      'scripts',
+      'wait-for-input.sh'
+    )
+
+    const _testExecOptions = getExecOptions()
+
+    _testExecOptions.listeners = {
+      stdout: (data: Buffer) => {
+        expect(data).toEqual(Buffer.from('this is my input\n'))
```

Comment:
This changes based on OS. On windows this should be `'this is my input\r\n'`

Reply:
Used `os.EOL`